### DISCUSSION TITLE
Discuss Clockwork V2 here

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#44] [General]
join the [Dis cord](http://discord.gg/Sb8NzVbqX8) for quicker responses from devs

**Comment by thrailkillr** [comment]
Doesn't work. Glitch is blocked at my school. Better stick with the old one.

**Comment by L413** [comment]
Im working on porting a ton more apps such as Cluster Rush (that one game where you jump on trucks), backrooms, doge miner, all fnaf games, and more!!!!!!!!!!

**Comment by eet14** [comment]
Please make it so that you can switch from this one to the old one. There is no themes and if you happen to factory reset it, you can't go back to the old themes and make it look like this.
![image](https://user-images.githubusercontent.com/126912952/222812159-fd2c89b3-bcf3-41f9-a5d0-555bfa4f473b.png)
![image](https://user-images.githubusercontent.com/126912952/222812330-a9eee361-65f4-48f8-b263-0e208ae801aa.png)

**Comment by eet14** [comment]
themes are gone 
![image](https://user-images.githubusercontent.com/126912952/222812638-cfd9c2c4-70ec-4b66-b7a3-af2de0850bc0.png)

**Comment by ToasterPanic** [comment]
@eet14 Please delete your cookies and reload the page.

**Comment by ToasterPanic** [comment]
At this point we aren't upgrading the app store for Clockwork V1 anymore (except for bug fixes)

The app store for Clockwork V2 already has most of the features planned for this implemented (and more,) just needs a bit of polishing and restyling.

You'll still be able to access a limited app store with verified apps (it won't require glitch access,) and you'll be able to manually install apps from a url in devtools.

**Comment by zpenguin360** [comment]
damn

**Comment by DiscipleOfJava** [comment]
So, Is there a confirmed release date for when V2 will release?

**Reply by ToasterPanic** [reply]
Should be some time in May/June. Not creating a release date until it's almost finished because I don't want the stress of deadlines on a project for fun.

**Comment by ToasterPanic** [comment]
join the [Dis cord](http://discord.gg/Sb8NzVbqX8) for quicker responses from devs
